client/src/visualiser-src/linked-list-visualiser/initialiser.ts

```diff
@@ -8,39 +8,40 @@ import { defaultSpeed } from './util/constants';
  */
 const initialiser = (): any => {
   const linkedList: LinkedList = new LinkedList();
-  const animationController = new LinkedListController();
+  const animationController = new Controller();
+
   animationController.setSpeed(defaultSpeed);
 
   const appendNode = (val: number, updateSlider: (val: number) => void): void => {
     animationController.finish();
     // Generating the steps of the animation
```

Comment:
Could probably do without these comments. Link to documentation?

Reply:
I feel like linking our documentation from confluence in source code might be a bit clunky especially if the links for documentation get changed

Reply:
I think it's okay. If we move the doc to other pages, then we can set up a redirect on Confluence's side I'm pretty sure.

https://confluence.atlassian.com/display/CONFKB/How+to+redirect+Confluence+users+to+a+new+URL